molecule/__init__.py

```diff
@@ -17,8 +18,13 @@
 #  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 #  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 #  DEALINGS IN THE SOFTWARE.
+"""Ansible-lint version information."""
 
-import pbr.version
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
 
-version_info = pbr.version.VersionInfo('molecule')  # noqa
-__version__ = version_info.release_string()
+try:
+    import pkg_resources
+    __version__ = pkg_resources.get_distribution('ansible-lint').version
```

Comment:
`%s/ansible-lint/ansible-molecule/`

Reply:
@decentral1se you know you can send Suggested Changes, right?
https://thenextweb.com/dd/2018/10/16/github-launches-suggested-changes-to-make-it-easier-to-collaborate-within-pull-requests/

P.S. It's just `molecule`, it's what is specified in dist metadata...

Reply:
I promise I tried clicking it a bunch of times but it wasn't working ... ;)